paradise_papers_django/search/models.py

```diff
@@ -25,12 +21,19 @@ class Entity(DjangoNode):
     Officers = RelationshipFrom('Officer', 'OFFICER_OF')
     Intermediaries = RelationshipFrom('Intermediary', 'INTERMEDIARY_OF')
     Addressess = RelationshipTo('Address', 'REGISTERED_ADDRESS')
-    Others = RelationshipTo('Other' , 'CONNECTED_TO')
+    Others = RelationshipFrom('Other' , 'CONNECTED_TO')
+    def Entities_relationship(self):
```

Comment:
Can you give me a little inside on this function? 😅

Reply:
The cypher query will return all/any relationship with another Entity, and the entity node which has the relationship. Then since it return an array I only need the node and the type of relationship and discard all other information. The list is returned so it could be implemented in the front end :D